service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -59,16 +59,29 @@ public Pipeline getPipeline(PipelinesEnum pipelineName) {
   }
 
   /**
-   * This method is meant to only be called by an admin endpoint to update the control workspace id
-   * for a pipeline
+   * This method is meant to only be called by an admin endpoint to update the control workspace id,
+   * workspace project, and/or workspace name for a pipeline. If any of the inputs are null, that
```

Comment:
the side effect of this strategy is it's impossible to nullify a populated value via this method. if we think we want this ability, we can change strategy a bit